components/charts/__tests__/__snapshots__/AdvancedTimeline-test.tsx.snap

```diff
@@ -39,8 +39,8 @@ exports[`should render correctly 1`] = `
           className="line-chart-grid"
           x1={0}
           x2={30}
-          y1={34.666666666666664}
-          y2={34.666666666666664}
+          y1={34.66666666666667}
```

Comment:
Interesting... 🤔

Reply:
My guess is that it's https://github.com/d3/d3-interpolate/issues/73 ?